crates/bevy_core_pipeline/src/bloom/downsampling_pipeline.rs

```diff
@@ -37,32 +25,41 @@ pub struct BloomUniforms {
     pub aspect: f32,
 }
 
-impl FromWorld for BloomDownsamplingPipeline {
-    fn from_world(world: &mut World) -> Self {
-        let render_device = world.resource::<RenderDevice>();
-
-        // Input texture binding
-        let texture = BindGroupLayoutEntry {
-            binding: 0,
-            ty: BindingType::Texture {
-                sample_type: TextureSampleType::Float { filterable: true },
-                view_dimension: TextureViewDimension::D2,
-                multisampled: false,
-            },
-            visibility: ShaderStages::FRAGMENT,
-            count: None,
+impl Mipmap for BloomDownsamplingMipmapper {
+    fn debug_names() -> &'static MipmapDebugNames {
```

Comment:
would be nice if we could do something like `MipmapDebugNames::with_prefix("bloom_downsampling")` and it automatically generated all the strings.